setup.py

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/env python
-from distutils.core import setup
-
-# hack for utf8 long_description support
+from setuptools import setup
 import sys
-reload(sys).setdefaultencoding("UTF-8")
+import setuptools
+
+version='0.8'
+EXTRAS_REQUIRE = {}
+INSTALL_REQUIRES = []
+
 
-version='0.7'
+if int(setuptools.__version__.split(".", 1)[0]) < 18:
+    if sys.version_info[0:2] < (3, 0):
+        INSTALL_REQUIRES.append("future")
```

Comment:
Never thought it can break something. Well as far as I remember I just added it for the `http.client` import to support `HTTPConnection` both on py2 and py3. 

Later you said we can in fact drop the `http.py` (`utils.py` in my case) completely in the favour of `requests`. Then we can drop the `future` dependency as well.